### PR TITLE
Removing references to rss feed

### PR DESCRIPTION
**Pull request by maisiembleton**
Removing references to the rss feed for the blog. There didn't appear to be many changes with this one as it's still tied to the content of the /blog page. 

📓 References: 
-
- Find the vimaly ticket [here](https://vimaly.com/#j8mqm/t/Removing_reference_to_blog_rss_feed_from_middleman/EGB2l9eghxysJOjxS)

### CODE REVIEW

File: source/layouts/partials/_meta.html.erb [hunk 1]
```diff
@@ -116,12 +116,11 @@
 <meta name="twitter:site:id" content="<%= config[:twitter_site_id] %>">
 
 <%# Links; next/prev should be canonical. %>
-<link rel="alternate" type="application/rss+xml" title="Sharesight RSS Feed" href="<%= base_url('/blog/feed.xml') %>">
```
Comment (kylorhall) [inline source/layouts/partials/_meta.html.erb:119]: I think you still need to link to this, even though the link no longer exists here…it exists in Gatsby.

File: spec/features/blog_categories_page_spec.rb [hunk 2]
```diff
@@ -45,7 +45,6 @@
         expect(page).to respond_successfully
 
         expect(page).to have_head('link', args: { rel: 'canonical', href: absolute_url(page_url) }, debug: :href)
-        expect(page).to have_head('link', args: { rel: 'alternate', href: base_url('blog/feed.xml') }, debug: :href)
```
Comment (kylorhall) [inline spec/features/blog_categories_page_spec.rb:48]: Restore this if you restore the link.

File: spec/features/blog_posts_page_spec.rb [hunk 3]
```diff
@@ -41,7 +41,6 @@
       expect(page).to have_head('link', args: { rel: 'canonical', href: URI::unescape(post.url) }, debug: :href)
       expect(page).to have_meta('og:url', URI::unescape(post.url), name_key: 'property')
 
-      expect(page).to have_head('link', args: { rel: 'alternate', href: base_url('blog/feed.xml') }, debug: :href)
```
Comment (kylorhall) [inline spec/features/blog_posts_page_spec.rb:44]: Restore this if you restore the link.

File: spec/features/landing_pages_page_spec.rb [hunk 4]
```diff
@@ -44,7 +44,6 @@ def expect_facebook_metas(page)
   def expect_urls(page, landing_page)
     expect(page).to have_head('link', args: { rel: 'canonical', href: landing_page[:url] }, debug: :href)
     expect(page).to have_meta('og:url', base_url(landing_page[:path]), name_key: 'property')
-    expect(page).to have_head('link', args: { rel: 'alternate', href: base_url('blog/feed.xml') }, debug: :href)
```
Comment (kylorhall) [inline spec/features/landing_pages_page_spec.rb:47]: Restore this if you restore the link.

**Review by kylorhall — CHANGES_REQUESTED** [review]
Removing the file LGTM…but I'm not sure sure about removing the `<link>` itself..

**Comment by maisiembleton** [comment]
Does that mean there's actually no changes to middleman for this page? @kylorhall

**Comment by kylorhall** [comment]
> Does that mean there's actually no changes to middleman for this page? @kylorhall

You should still delete the builder itself; Middleman shouldn't build this page anymore, but it still needs to link to it.

**Review by kylorhall — APPROVED** [review]
Looks fine…I'll just note all of your files aside from `source/blog/feed.xml.builder` are just whitespace changes—it's a bit weird 😛

Your IDE probably needs fixes if it's saving whitespace arbitrarily.

**Pull request merged**